MatchingGame.js

```diff
@@ -27,17 +25,17 @@ const getData = () => [
     { imgSrc: "images/Sasuke.jpeg", name: "Sasuke"},
 ];
 
-//Randomize
+// Randomize
 const randomize = () => {
     const cardData = getData();
     cardData.sort(() => Math.random() - 0.5);
     return cardData;
 };
 
-//Card Generator 
+// Card Generator 
 const cardGenerator = () => {
```

Comment:
I would rename this function that describes it better. If i read `cardGenerator` i think of a one-off utility function for generating a list of cards or something, not a function that is entrypoint for the game itself. Maybe something like `playCardGame` or `startCardGame` or `initAndPlayGame` etc.